main.go

```diff
@@ -206,6 +168,51 @@ func run(c *cli.Context) error {
 		secretTemplate = ".kube.sec.yml"
 	}
 
+	// Parse variables.
+	vars := make(map[string]interface{})
+	varsJSON := c.String("vars")
+	if varsJSON != "" {
+		if err := json.Unmarshal([]byte(varsJSON), &vars); err != nil {
+			return fmt.Errorf("Error parsing vars: %s\n", err)
+		}
+	}
+
+	// Parse secrets.
+	secrets := make(map[string]string)
+	for _, e := range os.Environ() {
+		if !strings.HasPrefix(e, "SECRET_") {
+			continue
+		}
+
+		// Only split up to 2 parts.
+		pair := strings.SplitN(e, "=", 2)
+
+		// Check that key and value both exist.
+		if len(pair) != 2 {
+			return fmt.Errorf("Error: secret key and value mismatch, expected 2, got %d", len(pair))
```

Comment:
Will this result in an error if you intentionally have a secret value equal to an empty string?